### ISSUE TITLE
Notification: notificationClass missing on NotificationSettings class

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-angular#1976]
## I'm submitting a...

* Suggestion for improvement  

## Current behavior
No `NotificationClass` property on `NotificationSettings` class. _Sorry for word pun_.

## Expected behavior
I would like to have same approach based on `PopupService` and `PopupSettings` class.
I found on `NotificationSettings` property like `width` and `height` but no css class.
Just see: https://www.telerik.com/kendo-angular-ui/components/popup/api/PopupSettings/#toc-popupclass to have a sample how i expected to find on Notification component.

## Environment

Package versions:

```
"@progress/kendo-angular-notification": "1.1.1",
"@progress/kendo-theme-default": "2.62.0",
```

**Comment by elena-gancheva** [comment]
This improvement is implemented and available in the latest `dev` version of the package.

**Comment by ipeshev** [comment]
[Working example](https://stackblitz.com/edit/angular-zimg5f?file=app/app.component.ts) .

**Comment by elena-gancheva** [comment]
Fixed in `v1.2.0` of the package.
